roles/lnls-ans-role-epics7/defaults/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
-epics7_base_version: 7.0.3
+epics7_base_version: 7.0.4.1
```

Comment:
why not move package version to group_vars/all like any other?

Reply:
We can do this, for sure. I will do it.